server/command.go

```diff
@@ -121,6 +284,31 @@ func (p *Plugin) executeCommandList(args *model.CommandArgs) {
 	for team := range teams {
 		str.WriteString(fmt.Sprintf("\n * %s", team))
 	}
+
+	// go through each key value pair and discern the teams with set values
+	page := 0
+	keys, err := p.API.KVList(page, 200)
+	if err != nil {
+		p.postCommandResponse(args, "Issue grabbing messages for teams.")
+		return
+	}
+	for len(keys) != 0 {
+		// retrieve id inside of kv pair
+		for _, key := range keys {
+			id := strings.ReplaceAll(key, welcomebotTeamWelcomeKey, "")
+			team, getTeamErr := p.API.GetTeam(id)
+			if getTeamErr != nil {
+				continue // the key is not corresponding to a team
+			}
```

Comment:
Can we check for the `welcomebotTeamWelcomeKey` prefix to ensure to only check these?

Reply:
I ended up removing this logic all together by retrieving a map with all the values beforehand. The new logic just loops through the map and using the key in each pair to append to the list of teams with welcome messages.